lib/language_pack/ruby.rb

```diff
@@ -920,6 +920,14 @@ def precompile_fail(output)
       msg << "Attempted to access a nonexistent database:\n"
       msg << "https://devcenter.heroku.com/articles/pre-provision-database\n"
     end
+
+    sprockets_version = bundler.gem_version('sprockets')
+    if output.match(/Sprockets::FileNotFound/) && (sprockets_version < Gem::Version.new('4.0.0.beta7') && sprockets_version > Gem::Version.new('4.0.0.beta4'))
+      msg << "If you have this file in your project\n"
```

Comment:
I would also suggest adding an `mcount` here :)

Reply:
I added an mcount, also standardized all the mcount names to not add "ruby" as a duplicate.